mkdocs.yml

```
@@ -125,6 +125,8 @@ edit_uri: https://github.com/hakancelikdev/unexport/tree/main/docs
 
 nav:
   - Home: index.md
+  - User Guide:
+      - tutorials/supported-behaviors.md
```

Can you change the tutorial file name?

```suggestion
      - tutorials/useful-features.md
```